controllers/topolvm_storageclass.go

```diff
@@ -93,7 +93,7 @@ func getTopolvmStorageClasses(lvmCluster *lvmv1alpha1.LVMCluster) []*storagev1.S
 	for _, deviceClass := range lvmCluster.Spec.DeviceClasses {
 		storageClass := &storagev1.StorageClass{
 			ObjectMeta: metav1.ObjectMeta{
-				Name: fmt.Sprintf("topolvm-%s", deviceClass.Name),
+				Name: fmt.Sprintf("odf-lvm-%s", deviceClass.Name),
```

Comment:
@nbalacha not sure it a problem for upgrades update of storageclass name is not supported we need to delete and recreate it?

Reply:
Upgrades are not in the picture as yet as this is the first downstream release.